bootstrap/service.go

```diff
@@ -321,18 +350,27 @@ func (bs bootstrapService) ChangeState(ctx context.Context, token, id string, st
 		return nil
 	}
 
+	channelsPage, err := bs.sdk.ChannelsByThing(cfg.ThingID, pm, token)
+	if err != nil {
+		return errors.Wrap(errChangeState, err)
+	}
+
+	for _, channel := range channelsPage.Channels {
+		existingChannels[channel.ID] = true
+	}
```

Comment:
I think we don't want this, because sending connect request to already connected channel should not throw error. 
If things API throw error. then we should think about changing the response of things API

Reply:
> I think we don't want this, because sending connect request to already connected channel should not throw error. If things API throw error. then we should think about changing the response of things API

@arvindh123 yes, things API throws `http status 409` when you send a connect request to an already connected channel.
`magistrala-things           | {"time":"2024-06-12T10:04:36.568921712Z","level":"WARN","msg":"Assign member to group failed to complete successfully","duration":"67.069682ms","group_id":"8fac693b-dc98-4178-b828-a4d4a8008d05","relation":"group","member_kind":"things","member_ids":["f2097dff-8e03-4461-92f1-80b972bb3c10"],"error":{"error":"entity already exists","message":"failed to add policies"}}`